src/test/test.ts

```diff
@@ -26,9 +26,15 @@ class Base {
   parent?: Suite;
 
   _only = false;
+  _tags: Set<string>;
 
   constructor(title: string) {
-    this.title = title;
+    this.title = title.split(' ').filter(x => !x.startsWith('#')).join(' ');
```

Comment:
Let's be more strict? `'#smoke #perf #fast Test something'.match(/((?:(?:#[^\s]+)[\s])+)(.*)*/)`